html/hillary.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+
+    <title>About Hillary</title>
+
+    <link href="https://fonts.googleapis.com/css?family=Lato|Rokkitt" rel="stylesheet">
+    <link rel="stylesheet" href="../css/bauer-style.css">
+  </head>
+
+  <body>
+
+    <div class="container">
+
+      <h1>Hillary Bauer</h1>
+      <h2>An Abridged History</h2>
+
+      <p>
+        I was born in Milford, MA.
+      </p>
+
+      <p>
+        I grew up.
+      </p>
+
+      <p>
+         After growing up, I lived a couple of other places, including Schenectady, NY where I went to Union College. I have a degree in Physics that I don't really use anymore; but I did make the background of this page space, so that's something.
+      </p>
+
+      <p>
+        I moved back to Boston a couple of years ago when my current company, ezCater, hired me. I started off writing copy for the site; but then I realized that I like coding, so I did a bunch of online tutorials, and now I work in direct support.
+      </p>
+
+      <p>
+        In my spare time, I decompress by watching sci-fi series while I knit things from those series. Recently, I started selling my stuff on <a href="https://www.etsy.com/shop/KnitInAMeeting/" target="_blank">Etsy</a> to fund my yarn habit.
+      </p>
+
+      <a href="https://github.com/bauer-hillary-c" target="_blank">
+        <img src="../images/github.png" atl="GitHub">
+      </a>
+
+      <a href="https://www.facebook.com/hillary.bauer.3" target="_blank">
+        <img src="../images/facebook.png" atl="Facebook">
+      </a>
+    </div><!-- .container --> 
```

Comment:
ah, i marked the top and bottom of me container div so that it was easier to see what was in there when i was styling it. just forgot to delete it